src/Controller/Admin/MeasuringUnitCrudController.php

```diff
@@ -15,6 +18,23 @@ public static function getEntityFqcn(): string
         return MeasuringUnit::class;
     }
 
+    public function configureActions(Actions $actions): Actions
+    {
+        return $actions
+            ->update(Crud::PAGE_INDEX, Action::NEW, function (Action $action) {
+                return $action->setLabel('measuring_unit.create_title');
+            })
+            ;
+    }
+
+    public function configureCrud(Crud $crud): Crud
+    {
+        return $crud
+            ->setEntityLabelInSingular('measuring_unit.singular')
+            ->setEntityLabelInPlural('measuring_units.plural')
+            ->setPageTitle('index', 'measuring_unit.list_title');
```

Comment:
plutôt utiliser une constante pour cibler la page à modifier

```suggestion
            ->setPageTitle(Action::INDEX, 'measuring_unit.list_title');
```